package.json

```diff
@@ -4,17 +4,20 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "test": "jest --watch --coverage=true --changedSince=origin/master -u"
```

Comment:
I've never seen these commands after --watch.  I like how it will make an update.